modules/programs/chromium.nix

```diff
@@ -19,10 +19,30 @@ let
 
       package = mkOption {
         inherit visible;
-        type = types.package;
+        type = types.nullOr types.package;
         default = defaultPkg;
         defaultText = literalExample "pkgs.${browser}";
-        description = "The ${name} package to use.";
+        description = ''
```

Comment:
I think that we should also add an assertion that if `commandLineArgs` is not `[]`, then `package` must be `null`

Reply:
I think you mean if `commandLineArgs`  is not `[]`, then `package` must not be `null`.

I see what you mean, but `commandLineArgs` function in the same way as the sway module in that the option is only used for the default package, and is essentially a no op for custom packages, which it is impossible to determine if such wrapper interface exists or provide the same function.

https://github.com/nix-community/home-manager/blob/master/modules/services/window-managers/i3-sway/sway.nix#L319-L324